src/controllers/sdex/index.ts

```diff
@@ -72,4 +73,46 @@ router.get(
   }),
 );
 
+router.get(
+  '/tickers',
+  asyncRoute(async (req, res) => {
+    const { chainId } = req.query;
+
+    const poolsResponse = await req.network.sdex.queryPools(1e3);
+    const volumeResponse = await prepareSdexVolume(req.network.chainId);
+
+    const volumeMap = new Map();
+    volumeResponse.forEach(({ token, volume }) => {
+      volumeMap.set(token.toLowerCase(), volume);
+    });
+
+    const tickers = poolsResponse.pools.map((pool) => {
+      const baseToken = pool.base.toLowerCase();
+      const quoteToken = pool.quote.toLowerCase();
+      const baseVolume = bignumber(volumeMap.get(baseToken) || '0');
+      const quoteVolume = bignumber(volumeMap.get(quoteToken) || '0');
+
+      const lastPrice = baseVolume.isZero() ? '0' : quoteVolume.div(baseVolume).toFixed(18);
+
+      const liquidityInUsd =
+        baseVolume.isZero() || lastPrice === '0' ? '0' : baseVolume.times(bignumber(lastPrice)).toFixed(18);
+
+      return {
+        ticker_id: `${pool.base}_${pool.quote}`,
+        base_currency: pool.base,
+        target_currency: pool.quote,
+        last_price: lastPrice,
+        base_volume: baseVolume.toString(),
+        target_volume: quoteVolume.toString(),
+        pool_id: pool.poolIdx.toString(),
+        liquidity_in_usd: liquidityInUsd,
+      };
+    });
```

Comment:
put this inside maybeCacheResponse callback, with current solution you still make all the requests, but are returning last saved data in the response for user